gulp/firebase-init.js

```diff
@@ -9,6 +9,7 @@ admin.initializeApp({
 });
 
 const db = admin.database();
+db.goOffline();
```

Comment:
Not having this was causing a hang in gulp tasks that did not make use of the firebase admin sdk.